frontend/javascripts/oxalis/view/left-border-tabs/histogram_view.tsx

```diff
@@ -199,7 +199,9 @@ class Histogram extends React.PureComponent<HistogramProps, HistogramState> {
   };
 
   tipFormatter = (value: number) =>
-    value > 10000 ? value.toExponential() : roundTo(value, this.getPrecision()).toString();
+    value >= 100000 || (value < 0.001 && value > -0.001)
```

Comment:
```suggestion
    value >= 100000 || (value < 0.001 && value > -0.001 && value !== 0)
```

otherwise, 0 renders as `0e+0` :)